raiden-cli/src/index.ts

```diff
@@ -1,206 +1,24 @@
-    transfer.changedAt.getTime() - transfer.startedAt.getTime(),
-    ', reveal =',
-    revealAt! - transfer.startedAt.getTime(),
-    ', unlock =',
-    unlockAt! - transfer.startedAt.getTime(),
-  );
-  return transfer;
-}
-
-async function raidenInit({
-  token,
-  partner,
-  deposit,
-  mint,
-  transfer: value,
-  target,
-}: {
-  token?: string;
-  partner?: string;
-  deposit?: BigNumberish;
-  mint?: boolean;
-  transfer?: string;
-  target?: string;
-} = {}) {
-  const start = Date.now();
-  // wait started
-  await raiden.action$.pipe(first((a) => a.type === 'matrixSetup')).toPromise();
-  await raiden.events$.pipe(first((a) => a.type === 'newBlock')).toPromise();
-  log.warn(
-    'Started: #',
-    await raiden.getBlockNumber(),
-    new Date(),
-    ', balance =',
-    (await raiden.getBalance()).toString(),
-    ', took',
-    Date.now() - start,
-  );
-
-  if (token) {
-    log.info('Token:', token, 'balance =', (await raiden.getTokenBalance(token)).toString());
-
-    if (partner) await raidenEnsureChannelIsOpen(token, partner, deposit, mint);
-
-    if (value && target) {
-      const availability = await raiden.getAvailability(target);
-      if (!availability.available) {
-        log.error('Target', target, 'not available:', availability);
-      } else {
-        await raidenTransfer(token, target, value);
-      }
-    }
-  }
-  const udcToken = await raiden.userDepositTokenAddress();
-  const amount = parseEther('10');
-  if ((await raiden.getTokenBalance(udcToken)).isZero()) {
-    await raiden.mint(udcToken, amount);
-    await raiden.depositToUDC(amount);
-  }
-  log.warn('Init: took', Date.now() - start);
-}
-
-async function setupApi(port: number) {
-  const api = '/api/v1';
-  app = express();
-  app.use(express.json());
-
-  app.get(`${api}/channels`, ({}, res) => {
-    raiden.channels$.pipe(first()).subscribe((c) => res.json(c));
-  });
-
-  app.get(`${api}/address`, ({}, res) => {
-    res.json({ our_address: raiden.address });
-  });
-
-  app.post(`${api}/payments/:token/:target`, (req, res) => {
-    raidenTransfer(req.params.token, req.params.target, req.body['amount'], req.body).then(
-      (tr) => res.json(tr),
-      (err) => res.status(400).json(err),
-    );
-  });
-
-  app.post(`${api}/config`, (req, res) => {
-    raiden.updateConfig(req.body);
-    res.json(raiden.config);
-  });
-
-  return new Promise<{ app: Express; server: Server }>(
-    (resolve) =>
-      (server = app.listen(port, () => {
-        log.info(`Serving Raiden LC API at http://localhost:${port}...`);
-        resolve({ app, server });
-      })),
-  );
-}
-
-async function main() {
-  logging.setLevel(logging.levels.DEBUG);
-  const argv = yargs
-    .usage('Usage: $0 -k <private_json_path> -e <node_url> --serve <port>')
+import { Raiden } from 'raiden-ts';
+import { CliArguments, Cli } from './types';
+import { makeCli } from './cli';
+import { setupLoglevel } from './utils/logging';
+
+const DEFAULT_RAIDEN_CONFIG = {
```

Comment:
Nitpick: it'd be interesting to have this in a `config.json` file, imported directly from typescript, like `import DEFAULT_RAIDEN_CONFIG from './config.json';` or something like that. Just a small QoL improvement ;)

Reply:
Yes of course. I love it. Always hated that this was in the code. I was just focusing on other stuff...

Reply:
:heavy_check_mark: